platform/pc/boot/stage1.s

```diff
@@ -75,11 +75,42 @@ dap:
         .lba          dq 1
 
 
+%ifdef DEBUG
```

Comment:
I missed this before. We don't actually want this assembled conditionally since we're using it for error output...right now the error report isn't getting built. You'll probably need to specify src/x86_64 in the nasm include path...or just spell it out here.